src/plugins/intel_gpu/src/graph/graph_optimizer/reorder_inputs.cpp

```diff
@@ -740,6 +740,41 @@ void reorder_inputs::run(program& p, reorder_factory& rf) {
         }
     };
 
+    const auto reorder_eltwise = [&p](typed_program_node<eltwise>& eltwise_node) {
```

Comment:
this function is getting too long. maybe could you make it as a separate function instead of lambda function?